dev/doc/INSTALL.make.md

```diff
@@ -119,6 +116,19 @@ please see the [contributing guide](../../CONTRIBUTING.md).
 
         make clean
 
+Notes for packagers
+-------------------
+
+The `make install` target for Coq's OCaml parts calls `dune
+install` internally. Before Dune 2.9, `dune install` didn't support
+configuring some installation paths such as `-docdir` and
+`-configdir`, thus these configure options were ignored by default.
```

Comment:
This note seems important enough to deserve a warning if the options are used in `configure`.

Reply:
Agree